### DISCUSSION TITLE
trying to use the {beta} Ifttt offline event notification

### DISCUSSION THREAD

**Discussion by kaunix** [mrlt8/docker-wyze-bridge#716] [Q&A]
I think I understand how to configure the yml: 
 you just add the following to the bottom of the .yml file: 
  -OFFLINE_IFTTT=MyEventName:[webhook_key_goes_here]

my question is, and admittedly this may be to the wrong group, 

how do I configure IFTTT to send an alert to my phone that this even has occurred?

also can I change "MyEventName" to something else like "DriveWayCamOffline"?

**Comment by mrlt8** [comment]
You'll need to give create an event name for the **IF** _Receive a web request_ and you can do a **Then** _Send a notification from the IFTTT app_ where you can style the message with the variables sent from the bridge.

As noted in the readme, the cameras will often show as "IOTC_ER_TIMEOUT" instead of offline. You could try to use that as a trigger instead of "IOTC_ER_DEVICE_OFFLINE" by setting:  `- OFFLINE_ERRNO=-13`

**Comment by kaunix** [comment]
Thanks for the response I’ll give it a try and let you know the results On Feb 10, 2023, at 16:19, mrlt8 ***@***.***> wrote:﻿
You'll need to give create an event name for the IF Receive a web request and you can do a Then Send a notification from the IFTTT app where you can style the message with the variables sent from the bridge.
As noted in the readme, the cameras will often show as "IOTC_ER_TIMEOUT" instead of offline. You could try to use that as a trigger instead of "IOTC_ER_DEVICE_OFFLINE" by setting:  - OFFLINE_ERRNO=-13

**Comment by kaunix** [comment]
I'm sure I'm hacking the hell out of this... I guess I just don't understand what variables need to go where: 
<img width="528" alt="If Maker Event Wyze_Camera_Offline, then Send a notification from 2023-02-10 17-48-47" src="https://user-images.githubusercontent.com/30199459/218235197-ac93b026-d36a-4ef5-8c63-e317c480f5e2.png">
<img width="401" alt="If Maker Event Wyze_Camera_Offline, then Send a notification from 2023-02-10 17-47-26" src="https://user-images.githubusercontent.com/30199459/218235223-c5caf6ff-1555-4382-b5f9-998cf6081f9d.png">
This is my docker-compose.yml
version: '2.4'
services:
    wyze-bridge:
        container_name: wyze-bridge
        restart: unless-stopped
        image: mrlt8/wyze-bridge:latest
        ports:
            - 1935:1935 # RTMP
            - 8554:8554 # RTSP
            - 8888:8888 # HLS
            - 8889:8889 #WebRTC
            - 8189:8189/udp # WebRTC/ICE
            - 5001:5000 # WEB-UI
        environment:
            # [REQUIRED] Fill the following two lines:
            - WYZE_EMAIL=<REDACTED>
            - WYZE_PASSWORD=<REDACTED>
            # [OPTIONAL] IP Address of the host to enable WebRTC e.g.,:
            - WB_IP=<REDACTED>
            # [OPTIONAL] Allow IFTTT to receive notifications of down cameras
            - OFFLINE_IFTTT=Wyze_Camera_Offline:<REDACTED>

**Reply by mrlt8** [reply]
`EventName` will be whatever you set in IFTTT. `Wyze_Camera_Offline` in your case.

the other values will be:

```json
  "value1" : "cam-name", //camera name
  "value2" : -90, // error no
  "value3" : "IOTC_ER_DEVICE_OFFLINE" // error name
```

**Reply by kaunix** [reply]
do they need to be included in the notification, I guess is my question?

 "{{Wyse_Camera_Offline}" is Offline. The {{cam-name}} is offline with an error of {{-90}} which is {{IOTC_ER_DEVICE_OFFLINE}} as of {{OccurredAt}}

**Reply by mrlt8** [reply]
No, but they are available if needed.
